typings/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@woocommerce/e2e-utils';
```

Comment:
Typescript was complaining when trying to import this library although the `tsconfig.json` has allowJs set to `true`.
I am not sure why that is the case, declaring it as a module here fixed it, I suppose we could create types for the library, as they don't have any for them.